cheroot/test/conftest.py

```diff
@@ -25,3 +30,76 @@ def wsgi_server_client(wsgi_server):
 def native_server_client(native_server):
     """Create a test client out of given HTTP server."""
     return get_server_client(native_server)
+
+
+@pytest.fixture
+def make_session_for_cheroot(wsgi_server):
```

Comment:
Can this be useful for end-users of cheroot in the testing of their apps? If yes, then it should go to `cheroot.testing` module.

Reply:
Well, `pyriform` is should help testing it without starting an HTTP server.

Reply:
My understanding is that this current fixture is for starting an HTTP server.

I had to replace certain parts of the TestApp and HostProxy in order to get this to work.  Also there is two sets of requests -- one used by HostProxy and one by Pyriform itself.

Reply:
Could probably think about how to add a Pyriform fixture for cherrypy.  Not sure how to add one for cheroot with a different framework.

Reply:
What do you mean by saying that? cheroot with a different framework?